pac4j-saml/src/main/java/org/pac4j/saml/sso/impl/SAML2DefaultResponseValidator.java

```diff
@@ -674,7 +674,12 @@ protected final void validateSignature(final Signature signature, final String i
     private boolean isDateValid(final DateTime issueInstant, final int interval) {
         final DateTime before =  DateTime.now().plusSeconds(acceptedSkew);
         final DateTime after =  DateTime.now().minusSeconds(acceptedSkew + interval);
-        return issueInstant.isBefore(before) && issueInstant.isAfter(after);
+        boolean isDateValid = issueInstant.isBefore(before) && issueInstant.isAfter(after);
+		if (logger.isTraceEnabled() && !isDateValid) {

```

Comment:
Yes, you should remove it since slf4j version 1.7 I think, but you also need to change the way you log things: from `logger.trace("myvalue is: " + myvalue)` to `logger.trace("myvalue: {}", myvalue);`. Following this rule, there will be no performance impact (no String building, these are just parameters).